maestro-android/src/androidTest/java/dev/mobile/maestro/ToastAccessibilityListener.kt

```diff
@@ -0,0 +1,62 @@
+package dev.mobile.maestro
+
+import android.app.UiAutomation
+import android.graphics.Rect
+import android.util.Log
+import android.view.accessibility.AccessibilityEvent
+import android.view.accessibility.AccessibilityNodeInfo
+import android.widget.Toast
+import kotlin.reflect.jvm.jvmName
+
+object ToastAccessibilityListener : UiAutomation.OnAccessibilityEventListener {
```

Comment:
That is pretty neat!

Reply:
Curious, are there any other events we can leverage here for other features?

Reply:
Hmm, I could give a try to see what it exposes. Good idea!